src/partials/footer.html

```diff
@@ -25,6 +25,8 @@
       <p class="footer__contact-title">Feel free to contact</p>
       <a class="footer__contact-phone" href="tel:+61(0) 383 766 284">+61(0) 383 766 284</a>
     </div>
-    <p class="footer__copy-rights">&#169; 2019 - IceCream.<br />All Rights Reserved.</p>
+    <div class="footer__copy-rights">
+      <p>&#169; 2019 - IceCream.<br />All Rights Reserved.</p>
```

Comment:
Hejka, świetna robota. Jedyne co bym zmieniła, to usunęła tag "br". Staramy się go w ogóle nie używać bez potrzeby. Tutaj możemy ładnie wybrnąć z sytuacji przy stylizacji nadając określoną szerokość diva ;)

Reply:
Zrobione, już wysyłam :)